paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/onramp/OnrampActivity.kt

```diff
@@ -63,12 +63,16 @@ internal class OnrampActivity : ComponentActivity() {
             registerUserCallback = viewModel::onRegisterUserResult
         )
 
-        onrampCoordinator = OnrampCoordinator.Builder(callbacks).build(this)
+        onrampCoordinator = OnrampCoordinator.Builder(
+            callbacks,
+            "pk_test_your_key_here",
+            "test_account_id"
+        ).build(this)
```

Comment:
we can keep this for now so that keys are available at injection time. I'll give it a better thought and we can iterate if needed.